pyvisa/__init__.py

```diff
@@ -23,8 +23,11 @@
 
 
 def log_to_screen(level=logging.DEBUG):
+    log(None, level) # sys.stderr by default
+
+def log(stream_output, level=logging.DEBUG):
```

Comment:
To avoid any confusion on what this function do I would prefer for it to be called `log_to_stream`. Otherwise people may think that it actually log a message.